cdap-master/src/test/java/io/cdap/cdap/master/environment/k8s/MasterServiceMainTestBase.java

```diff
@@ -129,6 +130,11 @@ public static void init() throws Exception {
     for (Class<? extends AbstractServiceMain> serviceMainClass : serviceMainClasses) {
       startService(serviceMainClass);
     }
+
+    // Creates default namespace from appfabric service required for tests.
+    getServiceMainInstance(AppFabricServiceMain.class).getInjector()
+        .getInstance(DefaultNamespaceCreator.class)
+        .execute(null);
```

Comment:
is it possible to load `AppFabricProcessor` service in tests so that `DefaultNamespaceCreator` will get executed automatically?